scripts/multinode/configure-sec.sh

```diff
@@ -89,7 +89,7 @@ function configure_kubelet() {
             exit 1
     esac
 
-    curl -sLO "https://dl.k8s.io/release/v1.32.3/bin/linux/${kube_arch}/kubelet" --output-dir "${KUBELET_HOME}"
+    curl -sLO "https://dl.k8s.io/release/v1.32.5/bin/linux/${kube_arch}/kubelet" --output-dir "${KUBELET_HOME}"
```

Comment:
You need to update hashes.
```bash
amd64 2b2988edd1646bf139dee6956d4283c520ff151a36febd10701ffda4852b8250
arm64 034753a2e308afeb4ce3cf332d38346c6e660252eac93b268fac0e112a56ff46
```
Sourced from
```bash
amd64 https://cdn.dl.k8s.io/release/v1.32.5/bin/linux/x86_64/kubelet.sha256
arm64 https://cdn.dl.k8s.io/release/v1.32.5/bin/linux/arm64/kubelet.sha256
```

Reply:
silly me :D